lib/ansible_tower_client/api.rb

```diff
@@ -35,6 +35,10 @@ def credentials
       Collection.new(self, credential_class)
     end
 
+    def credential_types
+      Collection.new(self, credential_type_class)
```

Comment:
so in a `v1` case, what is the expected behavior when accessing this?

Reply:
Since there is no such endpoint in v1, user will get 404 on http operations.
This is normal, because if user wants to work with v1, he should not use this object.